sonar-kotlin-api/src/main/java/org/sonarsource/kotlin/api/telemetry/TelemetryData.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * SonarSource Kotlin
+ * Copyright (C) 2018-2025 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the Sonar Source-Available License Version 1, as published by SonarSource SA.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the Sonar Source-Available License for more details.
+ *
+ * You should have received a copy of the Sonar Source-Available License
+ * along with this program; if not, see https://sonarsource.com/license/ssal/
+ */
+package org.sonarsource.kotlin.api.telemetry
+
+import org.sonar.api.batch.sensor.SensorContext
+
+class TelemetryData {
+    var hasAndroidImports = false
+    var hasAndroidImportsReportedAsTrue = false
```

Comment:
Used to stop reporting `hasAndroidImports` once it is reported as `true`.
This is relevant in multi-module projects, where the `Sensor` is executed more than once (i.e. in Scanner for Gradle and Maven contexts).